javascript/first-js-oop-project.md

```diff
@@ -113,10 +134,11 @@ This mini project will help you to learn basic concepts about objects in JavaScr
 
 
 1. Once you are done with the above code, you will get an unordered list of all the events along with the descriptions in the web page as the output of your code. The output on the web page should show the following:
-     - KLOS Golden Gala - An evening with hollywood vampires
-     - Skillet & Sevendust - Victorious war tour
-     - Jenny Lewis - On the line tour 2019
+- KLOS Golden Gala - An evening with hollywood vampires - Eligible Tickets: 1. vampire
+- Skillet & Sevendust - Victorious war tour - Eligible Tickets: 1. General Admission 2. Floor Seating
+- Jenny Lewis - On the line tour 2019 - Eligible Tickets: 1. Balcony
 
 ### Independent Practice
 
+- Update the jquery line that specifies price range to ensure that your function works for edge cases
```

Comment:
More ideas for independent practice:
- Add a UI that lets the user specify their price range
- Add another function to your event that shows the cheapest ticket for the event, and display them on the HTML page